packages/start/root/FileRoutes.tsx

```diff
@@ -1,15 +1,17 @@
 /// <reference path="../server/types.tsx" />
 
 // @ts-expect-error
-var routes = $ROUTES;
+var routeData = $ROUTE_DATA;
```

Comment:
nit: Can we change $ROUTE_DATA to $ROUTES_CONFIG, since RouteData already means something else in solid-start, doesnt help to overload the term.. and the singular ROUTE_DATA doesnt communicate the meaning properly..

Reply:
You are right I struggled with coming up with a good name for that one.

Your change has been applied.